unittest.cfg

```diff
@@ -1,13 +1,19 @@
 [unittest]
 verbose = 2
-plugins = flufl.testing.nose
+plugins =
+    nose2.plugins.prof
 
 [log-capture]
 always-on = False
+clear-handlers = True
+date-format = None
+; format = %%(name)s: %%(levelname)s: %%(message)s
+format = %%(asctime)s: %%(name)s: %%(levelname)s: %%(message)s
+filter = -nose
+log-level = DEBUG
 
-[flufl.testing]
-always-on = True
-package = aiosmtpd
-setup = aiosmtpd.testing.helpers.setup
-teardown = aiosmtpd.testing.helpers.teardown
-start_run = aiosmtpd.testing.helpers.start
+[profiler]
+always-on = False
+filename = _dynamic/cPROFILE
+restrict =
+sort = ncalls
```

Comment:
The `[profiler]` section replaces the `[flufl.testing]` section, containing some customizations:

* `always-on = False` because we want to selectively enable profiling only for certain testenvs
* `filename = _dynamic/cPROFILE` move profiling file into an ignored directory (see `.gitignore`) to prevent messing up `git status`
* `sort = ncalls` temporary setting to help ensure whatever being called the most is highly optimized.